### ISSUE TITLE
Nothing happen when I try to run the model using the pretrained weights.

### ISSUE THREAD

**Issue by imaginistLi** [bardiadoosti/HOPE#6]
I downloaded the dataset and generated the `.npy` files, and then, I downloaded the pretrained weights file `model-0.pkl` to run the testing code. But,  when I was running this command `python HOPE.py --input_file ./datasets/fhad/ --test --batch_size 64 --model_def HopeNet --gpu --gpu_number 0 --pretrained_model ./checkpoints/fhad/model-0.pkl`, nothing happened, I can only see this in the terminal,
![image](https://user-images.githubusercontent.com/54342550/81884760-495db880-95cb-11ea-8407-49ef9411335b.png)

**Comment by imaginistLi** [comment]
Is there anybody can help me?

**Comment by bardiadoosti** [comment]
It seems that your numpy files are empty. Will you please post your numpy arrays shapes?

**Comment by imaginistLi** [comment]
Oh yes, it is empty. It seems that something wrong when I was making the `.npy` files, let me try again.

**Comment by imaginistLi** [comment]
Thank you! problem solved.
